packages/integration-sdk-cli/src/commands/types.ts

```diff
@@ -0,0 +1,154 @@
+import * as log from '../log';
+import { loadConfig } from '../config';
+import * as path from 'path';
+import { buildStepDependencyGraph } from '@jupiterone/integration-sdk-runtime';
+import { createCommand } from 'commander';
+import {
+  IntegrationStepExecutionContext,
+  Step,
+  StepGraphObjectMetadataProperties,
+  StepEntityMetadata,
+  StepRelationshipMetadata,
+} from '@jupiterone/integration-sdk-core';
+import { promises as fs } from 'fs';
+
+export interface TypesCommandArgs {
+  projectPath: string;
+  typesFilePath: string;
+}
+
+export function types() {
+  return createCommand('types')
+    .description('Collects types metadata for all steps')
+    .option(
+      '-p, --project-path <directory>',
+      'Absolute file path to the integration project directory. Defaults to the current working directory.',
```

Comment:
Nit, I'd drop "file path" for just "path", since it is a directory and not a file.
```suggestion
      'Absolute path to the integration project directory. Defaults to the current working directory.',
```